doc/BoxPlatform.md

```diff
@@ -1,71 +1,133 @@
+Typically this is done by passing a JSON Web Token in the Authorization header of the request to your remote server
+that is signed by your authentication system.  That way you can verify that the incoming request is indeed originating
+from an authorized app and device.
 
-```objectivec
-#import "AppDelegate.h"
+Below is an example of using a `BOXContentClient` instance to work with a specific `uniqueID`, including example code of
+how to retrieve a new token when the SDK detects a token expiration. Note that you are not required to specify an
+initial token as the SDK will simply invoke your `fetchTokenBlock`.
 
-@import BoxContentSDK;
+__Objective-C:__
+```objectivec
+ServerAuthFetchTokenBlock fetchTokens = ^(NSString *uniqueId, NSDictionary *fetchTokenBlockInfo, void (^completion)(NSString *, NSDate *, NSError *))
+{
+    // NOTE: This code is specific to your app; it is responsible for retrieving new tokens from a secure server
+    NSLog(@"Attempting to retrieve new access token from server for uniqueId: %@", uniqueId);
 
-@interface AppDelegate ()
+    // Make a request to the secure sever for a token
+    NSString *fullUrl = [NSString stringWithFormat:@"https://YOUR_SECURE_SERVER.example.com/token?id=%@", uniqueId];
+    NSURL *url = [NSURL URLWithString:fullUrl];
 
-@property (nonatomic, readonly, strong) BOXContentClient *boxClient;
+    __block NSString *token;
+    NSURLSessionDataTask *task = [[NSURLSession sharedSession] dataTaskWithURL:url
+            completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+                if (error) {
+                    NSLog(@"Failed to retrieve new access token from server");
+                    completion(nil, nil, error);
+                } else {
+                    token = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+                    NSLog(@"Succeeded in retrieving new access token: %@", token);
 
-@end
+                    // Pass token and expiration time back to the client
+                    completion(token, [NSDate dateWithTimeIntervalSinceNow:2700], nil);
+                }
+            }];
+    [task resume];
+};
 
-@implementation AppDelegate
+// Create a client for the authenticated user; the client is keyed to that user's unique ID for caching purposes
+// The unique ID can be any identifier for the user, or a dummy value
+ServerAuthUser *user = [[ServerAuthUser alloc] initWithUniqueID:@"UNIQUE_ID"];
+BOXContentClient *boxClient = [BOXContentClient clientForServerAuthUser:user
+                                                initialToken:nil
+                                                fetchTokenBlockInfo:nil
+                                                fetchTokenBlock:fetchTokens];
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+// Example of making an API call
+BOXUserRequest *request = [boxClient currentUserRequest];
```

Comment:
Do app users generally have a valid response to `/users/me`? I would think not. I know it's just an example.

Reply:
App Users should have a valid response to `GET /users/me`, provided that their token isn't downscoped